libraries/eddystone-advertising-library/.jshintrc

```diff
@@ -0,0 +1,9 @@
+{
+  "indent"  : 2,         // Number of spaces to use for indentation
+  "quotmark": "single",  // Require single quotes
```

Comment:
FYI I wondered if this was a typo and looked it up. Now I see it's not (!) but that it's deprecated and will be removed in the next release of jshint.